### ISSUE TITLE
Valid values for type parameter to twisted.conch.ssh.keys.Key.toString can't be determined except by inspecting private implementation details

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3689]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3689|
|Type|defect|
|Created|2009-03-12 21:19:22Z|
|Branch|https://github.com/twisted/twisted/tree/tostring-docs-3689|

`Key.toString` accepts `'OPENSSH'`, `'LSH'`, or `'AGENTV3'`.  There's no way to find this out, though, except by looking up the `_toString_*` helper methods.  The docstring for `Key.toString` should spell out possible values.

<details><summary>Searchable metadata</summary>

```
trac-id__3689 3689
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_tostring_docs_3689 branches/tostring-docs-3689
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__documentation documentation
time__1236892762000000 1236892762000000
changetime__1237570931000000 1237570931000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [26371]) Branching to 'tostring-docs-3689'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [26372]) expand Key.toString docs

refs [#3689](https://github.com/twisted/twisted/issues/3689)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/builders/documentation/builds/330)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @exarkun|
|-|-|

Looks good to me.  Merge away.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [26481]) Merge tostring-docs-3689

Author: exarkun
Reviewer: glyph
Fixes: [#3689](https://github.com/twisted/twisted/issues/3689)

Improve the docstring for `twisted.conch.ssh.keys.Key.toString`, primarily by
including the supported values of the `type` parameter.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
